public/css/header.css

```diff
@@ -130,65 +131,75 @@ img.avatar {
   visibility: visible;
 }
 
-.fxa-menu-links a {
+.fxa-menu-link {
   padding: 1rem 2rem 1rem 1rem;
   border-bottom: 1px solid rgba(238, 238, 238, 0.502);
   color: rgba(21, 20, 23, 0.77);
   background-color: rgba(255, 255, 255, 0);
 }
 
-.fxa-menu-links a:hover {
+.fxa-menu-link:focus,
+.fxa-menu-link:active {
+  background-color: rgba(238, 238, 238, 0.502);
+  border-bottom: 1px solid rgba(238, 238, 238, 0.502);
```

Comment:
We use this `rgba()` value at least 3 times here. Not sure if it's worth extracting out into a CSS variable, or if that is overkill.

Reply:
I think it's used elsewhere also so a variable is probably not overkill.